compute_endpoint/globus_compute_endpoint/endpoint/config/user_config_template.yaml

```diff
@@ -1,4 +1,4 @@
-# This is the default user-template provided with newly-configured Multi-Tenant
+# This is the default user-template provided with newly-configured Multi-user
```

Comment:
Should case match here? (There's another location in the code that has this same de-capitalization change.)

```suggestion
# This is the default user-template provided with newly-configured Multi-User
```

Reply:
Good catch - fixed